hardhat.config.js

```diff
@@ -21,12 +21,12 @@ const argv = require('yargs/yargs')()
       alias: 'enableGasReport',
       type: 'boolean',
       default: false,
+      implies: 'gasReport',
     },
     gasReport: {
       alias: 'enableGasReportPath',
       type: 'string',
-      implies: 'gas',
-      default: undefined,
+      default: 'gasReporterOutput.json',
```

Comment:
you are confsing two things:
- the json report that is always written to `gasReporterOutput.json`
- the human readable table, that is either in the terminal, or in a file.

This the the later. If we set that to `gasReporterOutput.json`, then we risk override the json file with non machine readable data.

We need to keep that undefined by default

Reply:
let me propose something

Reply:
See: 50d7935

Reply:
This looks waay better. It was kinda obvious sorry 😅 .

I wonder how many other things we're leaving around that could be simplified.